helper-cli/src/main/kotlin/utils/PathExcludeGenerator.kt

```diff
@@ -92,6 +134,20 @@ private fun File.getAncestorFiles(): List<File> {
     return result
 }
 
+private fun getClosestCommonAncestor(files: Collection<File>): File {
```

Comment:
I admit I haven't carefully check what *exactly* this does, but it sounds like our existing `getCommonFileParent` utility function, or?

Reply:
I just tried out `getCommonFileParent()`. It doesn't work as-is, because it turns the paths into absolute paths.
The functions in this file only use relative paths.

Reply:
I can look into changing `getCommonFileParent()` in that regard, then we could eventually simplify this later.

Reply:
Thanks!